documentation/source/users/rmg/installation/dependencies.rst

```diff
@@ -59,8 +19,3 @@ License Restrictions on Dependencies
 All of RMG's dependencies except the ones listed below are freely available and compatible with RMG's open source MIT license (though the specific nature of their licenses vary). 
 
 * **pydas**: The DAE solvers used in the simulations come from `Linda Petzold's research group <https://cse.cs.ucsb.edu/software/>`_ at UCSB.  For running sensitivity analysis in RMG, the DASPK 3.1 solver is required, which "is subject to copyright restrictions” for non-academic use. Please visit their website for more details. To run RMG without this restriction, one may switch to compiling with the DASSL solver instead in RMG, which is "available in the public domain.”
-
-If you wish to do on-the-fly quantum chemistry calculations of thermochemistry (advisable for fused cyclic species in particular, where the ring corrections to group additive estimates are lacking),
-the then you will need the third-party software for the QM calculations:
-
-* **gaussian**: Gaussian03 and Gaussian09 are currently supported and commercially available.  See `https://gaussian.com <https://gaussian.com>`_ for more details.
```

Comment:
Why was this removed?

Reply:
My initial thoughts were that this is out of date and needs to be updated. I also had the impression people aren't actively using it / it may be broken (see https://github.com/ReactionMechanismGenerator/RMG-Py/issues/1864), so to be honest I'm not sure what the current best practices for the QMTP module are. That said, on second thought we should definitely mention/include it here. latest changes include an attempt at revising this

Reply:
From the PR above, it looks like it's not quite as necessary for fused rings, so I suggest softening the language